release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -640,6 +640,20 @@ This is an alternative to explicitly setting an ASN in the `spec.peerASN` field.
 With this release, you can configure a Remote Direct Memory Access (RDMA) Container Network Interface (CNI) on Single Root I/O Virtualization (SR-IOV) to enable high-performance, low-latency communication between containers.
 When you combine RDMA with SR-IOV, you provide a mechanism to expose hardware counters of Mellanox Ethernet devices to be used inside Data Plane Development Kit (DPDK) applications.
 
+[id="ocp-release-notes-networking-aws-byo-static-eip-support-ingress_{context}"]
+==== Support for {aws-short} static IP addresses in the Ingress Controller
+
+With this release, you can now specify static IP addresses, or {aws-short} Elastic IP (EIP) addresses, for your Network Load Balancer (NLB) in the Ingress Controller.
+
+For more information, see xref:../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc#nw-ingress-aws-static-eip-nlb-configuration[Configuring {aws-short} Elastic IP (EIP) addresses for a Network Load Balancer (NLB)].
+
+[id="ocp-release-notes-networking-openstack-floatingip-support-ingress_{context}"]
+==== Support for pre-created {rh-openstack} floating IP addresses in the Ingress Controller
```

Comment:
Is there a release note section for OpenStack?  This seems more appropriately placed there, since it only applies to OpenStack.

Reply:
no, they have their own release notes. but we occasionally publish release notes when something is offered in OCP for specific platforms. going through some old OCP release notes networking section, we've done this for Nutanix, IBM Cloud, vSphere, AWS and even OpenStack from what I see (4.12 and 4.13)